### ISSUE TITLE
Dataloader

### ISSUE THREAD

**Issue by jizongFox** [MIC-DKFZ/batchgenerators#38]
Hi, 
Thanks for this great work. I saw that you used multithread to build a high-performance data loader. What would be its advantage over Framework's Dataloader?

**Comment by FabianIsensee** [comment]
From the documentation:

> We supports a variety of augmentations, all of which are compatible with 2D and 3D input data! (This is something that was missing in most other frameworks).

I assume by dataloader you mean the pytorch dataloader? As far as I know that one does not support 3D images. Also, we started developing batchgenerators a LONG time ago. Back then pytorch was not born yet :-)
Performance wise it is the same as the pytorch dataloader.

Best,
Fabian

**Comment by jizongFox** [comment]
Thanks for your comment and this wonderful job!
